src/main/webapp/dev/js/common/questionRubric.es6

```diff
@@ -10,6 +10,110 @@ import {
     disallowNonNumericEntries,
 } from './ui.es6';
 
+function getRubricChoiceElem(questionNum, col) {
+    return $(`#rubricChoice-${questionNum}-${col}`);
+}
+
+function getRubricWeightElem(questionNum, col) {
+    return $(`#rubricWeight-${questionNum}-${col}`);
+}
+
+function getRubricDescElem(questionNum, row, col) {
+    return $(`#rubricDesc-${questionNum}-${row}-${col}`);
+}
+
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
+    const $swapCell = $(`#rubric-options-row-${questionNum} .rubricCol-${questionNum}-${colIndex}`);
+    const rubricCellSelector = `td[class*='rubricCol-${questionNum}']`;
+
+    if ((isSwapLeft && $swapCell.prev(rubricCellSelector).length === 0)
+        || (!isSwapLeft && $swapCell.next(rubricCellSelector).length === 0)) {
+        // trying to swap left most or right most column
+        return;
+    }
+
+    const swapValues = ($a, $b) => {
+        const temp = $a.val();
+        $a.val($b.val());
+        $b.val(temp);
+    };
+
+    const numberOfRows = parseInt($(`#rubricNumRows-${questionNum}`).val(), 10);
+    let swapColIndex;
+
+    if (isSwapLeft) {
+        swapColIndex = $swapCell.prev(rubricCellSelector).attr('data-col');
+    } else {
+        swapColIndex = $swapCell.next(rubricCellSelector).attr('data-col');
+    }
+
+    // swap rubric choices
+    let $currentValue = getRubricChoiceElem(questionNum, colIndex);
+    let $swapValue = getRubricChoiceElem(questionNum, swapColIndex);
+    swapValues($currentValue, $swapValue);
```

Comment:
This is pretty much what you suggested in your review comment just above - 
``` js
const swap = (elemSelector, questionNum, indexOfFirstCol, indexOfSecondCol) => {
    const $a = elemSelector(questionNum, indexOfFirstCol);
    const $b = elemSelector(questionNum, indexOfSecondCol);
    const $temp = $a.val();
    $a.val($b.val());
    $b.val($a.val());
};
```

I am going with this.